docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Teleport Kubernetes Operator (Preview)
+description: Easily manage Teleport resources from Kubernetes
+---
+
+<Admonition type="warning" title="Preview">
+  Teleport Kubernetes Operator is currently in Preview mode.
+</Admonition>
```

Comment:
Nit: I don't know if we need to include this admonition if the title already says "preview". I'm not sure we've done this for other preview features since introducing the new product preview policy.

@ptgott @xinding33 Not sure if you folks have an opinion on this.

Reply:
I copied this from another preview page: pages/database-access/guides/sql-server-ad.mdx